Resources/Textures/Objects/Devices/singularity_beacon.rsi/meta.json

```diff
@@ -0,0 +1,20 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-3.0",
+  "copyright": "tgstation at c46d689e2b300bdc8c5b153855b1a5bbb2c5b168",
```

Comment:
Taken from tgstation at <permalink of dmi file>

Reply:
all other copyrights are commit hash

Reply:
commit hash is enough but its 10000% preferred if you use a permalink, please do